lib/pharos/phases/base.rb

```diff
@@ -21,8 +21,7 @@ def exec_script(script, vars = {})
 
       def parse_resource_file(path, vars = {})
         path = File.realpath(File.join(__dir__, '..', 'resources', path))
-        yaml = File.read(path)
-        Pharos::Erb.new(yaml).render(vars)
+        Pharos::YamlFile.new(path).read(vars)
```

Comment:
The `Pharos::YamlFile#read` is entirely generic, and not actually YAML-specific, so this works nicely for any "resource" files...

Should probably be renamed to something like `upload_config_file`, using `lib/pharos/configs` or such... the `resources` are for Kube resources.

Reply:
Maybe open an issue about this?

Reply:
#204